src/backend/distributed/sql/udfs/citus_tables/10.0-1.sql

```diff
@@ -18,3 +18,5 @@ WHERE
   partkey IS NOT NULL OR repmodel = 't'
 ORDER BY
   logicalrelid::text;
+
+GRANT SELECT ON public.citus_tables TO public;
```

Comment:
I think we should do this in seperate migration file (`10.0-2.sql`) since it is possible that someone already installed citus 10.0.0 from tag and they won't be able to get this change when doing `ALTER citus UPDATE`

Reply:
(Obviously this would require adding a new migration file too)

Reply:
It's also possible someone already installed master. I think we should only consider releases.

Reply:
since we merged https://github.com/citusdata/citus/pull/4747, let's have this change in `10.0-2.sql` and include it in `10.0-1--10.0-2` installation script

Reply:
(it might not add much value but we need to consider `downgrade` logic too)